index.html

```diff
@@ -61,7 +61,169 @@ <h3 class="third-heading">Logo</h3>
                         </div>
                     </div>
 
-                    
+                    <section class="main-content">
+                        <div class="home__page--img">
+                            <img class="home-bg" src="/assets/images/bg2.jpg" alt="Background image">
+                            <h1 class="company-Info first-heading">Creative Kids</h1>
+                            <div class="welcomeMsg">
+                                <h2 class="second-heading">upto 50% off</h2>
+                                <button class="shopNow-btn"><a href="/components/productsListPage/productList.html" target="_blank"
```

Comment:
Remove the `_blank` from target
These pages should open on the same page

Reply:
Okay